### PR TITLE
Example scenario - HPC SaaS

### PR DESCRIPTION
**Pull request by alexbuckgit**

### CODE REVIEW

File: docs/example-scenario/apps/hpc-saas.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: Computer-aided engineering through high performance computing on Azure
```
Comment (adamboeglin) [inline docs/example-scenario/apps/hpc-saas.md]: Let's work on this title.  Does this actually reflect the article?  Would it be better to call it a "Managed system for Computer Aided Design on Azure"?

File: docs/example-scenario/apps/hpc-saas.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+description: <Article Description>
+author: alexbuckgit
+ms.date: 08/22/2018
+---
+
+# Computer-aided engineering through high performance computing on Azure
+
+This example scenario demonstrates delivery of a software-as-a-service (SaaS) platform built on the high-performance computing (HPC) capabilities of Azure. This scenario is based on an engineering software solution. However, the architecture is relevant to other industries requiring HPC resources such as image rendering, complex modeling, and financial risk calculation.
```
Comment (adamboeglin) [inline docs/example-scenario/apps/hpc-saas.md]: Is this SaaS for internal resources or building a platform to be consumed by external customers?

File: docs/example-scenario/apps/hpc-saas.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+* Computational chemistry applications
+
+## Architecture
+
+![Architecture for a SaaS solution enabling HPC capabilities][architecture]
+
+* Users can access NV-series virtual machines with powerful GPUs for rendering and collaborative tasks via a browser with a custom Remote Desktop browser control. Users can edit their designs and view their results. The HPC scheduler spins up additional nodes based on the number of waiting users.
+* Users access a web application hosted in CentOS virtual machines to submit workloads to a queue for execution on available HPC cluster nodes.
+* Complex workloads are executed using nodes in an HPC compute cluster. The HPC scheduler invokes Resource Manager templates to spin up additional nodes based on the depth of the queue.
+* Simpler workloads are executed using an Azure Kubernetes Service cluster.
```
Comment (adamboeglin) [inline docs/example-scenario/apps/hpc-saas.md]: How is complex vs simple defined and routed?

File: docs/example-scenario/apps/hpc-saas.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+Other scenarios using this architecture might include:
+
+* Genomics research
+* Weather simulation
+* Computational chemistry applications
+
+## Architecture
+
+![Architecture for a SaaS solution enabling HPC capabilities][architecture]
+
```
Comment (adamboeglin) [inline docs/example-scenario/apps/hpc-saas.md]: Let's re-word this to sound more like a data flow.

File: docs/example-scenario/apps/hpc-saas.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+* [General purpose virtual machines](/azure/virtual-machines/linux/sizes-general) running CentOS handle more traditional workloads such as web applications.
+* [Application Gateway](/azure/application-gateway/) load balances the requests coming into the web servers.
+* [Azure Kubernetes Service (AKS)](/azure/aks/) is used to run scalable workloads at a lower cost for simulations that don't require the high end capabilities of HPC or GPU virtual machines.
+* [Altair PBS Works Suite](https://www.pbsworks.com/PBSProduct.aspx?n=PBS-Works-Suite&c=Overview-and-Capabilities) orchestrates the HPC workflow, ensuring that enough virtual machine instances are available to handle the current load. It also deallocates virtual machines when demand is lower to reduce costs.
+
+### Alternatives
+
+* [Azure CycleCloud](/azure/cyclecloud/overview) simplifies creating, managing, operating, and optimizing HPC clusters. It offers advanced policy and governance features. CycleCloud supports any job scheduler or software stack.
```
Comment (adamboeglin) [inline docs/example-scenario/apps/hpc-saas.md]: Would this replace Altair PBS Works suite?

File: docs/example-scenario/apps/hpc-saas.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+* [HPC Pack](/azure/virtual-machines/windows/hpcpack-cluster-options) can create and manage an Azure HPC cluster for Windows Server-based workloads. HPC Pack isn't an option for Linux-based workloads.
+* [Azure Automation State Configuration](/azure/automation/automation-dsc-overview) provides an infrastructure-as-code approach to defining the virtual machines in a cluster. When a new set of virtual machines is needed, they are provisioned using the latest definition specified in a PowerShell DSC configuration script.
+
+## Considerations
+
+* While using an infrastructure-as-code approach is a great way to manage virtual machine build definitions, it can take a long time to provision a new virtual machine using a script. This solution found a good middle ground by using the DSC script to periodically create a golden image, which can then be used to provision a new virtual machine much more quickly.
```
Comment (adamboeglin) [inline docs/example-scenario/apps/hpc-saas.md]: What's a DSC Script?

File: docs/example-scenario/apps/hpc-saas.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+* Balancing overall solution costs with fast availability of compute resources is a key consideration. Provisioning a pool of virtual machine instances and putting them in a deallocated state lowers the operating costs. When an additional virtual machine is needed, reallocating an existing instance is much faster than provisioning a new instance.
+* The original architecture relied entirely on Azure virtual machines for running simulations. In order to reduce costs for workloads that didn't require all the capabilities of a virtual machine, these workloads were containerized and deployed to Azure Kubernetes Service (AKS).
+* The company's workforce had existing skills in open source technologies. They can take advantage of these skills by building on technologies like Linux and Kubernetes. 
+
+## Pricing
+
+The following considerations will drive a substantial portion of the costs for this solution.
```
Comment (adamboeglin) [inline docs/example-scenario/apps/hpc-saas.md]: Can we add a link to the calculator with at least a stub ($0/$1 item) for every Azure service used?

**Comment by alexbuckgit** [comment]
Closing PR in favor of new PR targeting master.

**Pull request closed without merging**